scripts/config.pl

```diff
@@ -34,7 +36,7 @@
 #   MBEDTLS_REMOVE_3DES_CIPHERSUITES
 #   MBEDTLS_SHA256_NO_SHA224
 #   MBEDTLS_SSL_HW_RECORD_ACCEL
-#   MBEDTLS_SSL_PROTO_NO_DTLS
+#   MBEDTLS_SSL_PROTO_NO_TLS
```

Comment:
What does this flag mean ?

Reply:
This is just some clean up work in the comments.